bourbonandburial/BourbonAndBurial
# Ml edit profile

Closes #66 

How to test:
- log in
- click on user's profile
- click the edit icon. 
      - Modal appears
      - Form is prepopulated with user's info 
      - isEditing state is true.
- click the cancel button on modal
      - isEditing state is false
- click the edit button
- update the form
- click submit
      - profile is updated
      - navbar is also updated
